app/Http/Controllers/PageController.php

```diff
@@ -139,8 +140,12 @@ public function api_show(Page $page)
      */
     public function edit($id)
     {
+        if(!Gate::allows('web')){
+            abort(404, "Access denied");
+        }
```

Comment:
Whats your goal with this?

Reply:
Just trying to use Gate to restrict updating access, but we did not use it. I Removed it now.